gb.yaml

```
@@ -507,6 +513,21 @@ tests:
       regions: ["gb"]
     expect:
       name: "Platinum Jubilee"
+  - given:
+      date: '2022-09-18'
```

Should this be the 19th instead?
```suggestion
      date: '2022-09-19'
```

---

Yes, you are right... Fixed!

Actually, that means (checks...) ... that `make test` doesn't run the `expect` sections?

---

No, the tests are run within a separate repo (https://github.com/holidays/holidays ) which imports the definitions and the tests.